server/views/partials/layout.njk

```diff
@@ -3,13 +3,6 @@
 
 {% block head %}
   <link href="/assets/stylesheets/application.css?{{ version }}" rel="stylesheet"/>
-
-  <script src="/assets/js/jquery.min.js"></script>
-  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"
-          integrity="sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU="
-          crossorigin="anonymous"></script>
-  <link href="https://code.jquery.com/ui/1.12.1/themes/ui-lightness/jquery-ui.css" rel="stylesheet">
-
```

Comment:
This probably means that the CSP (Content Security Policy) could be simplified:

https://github.com/ministryofjustice/manage-my-prison/blob/main/server/middleware/setUpWebSecurity.ts

Reply:
ah thanks, i missed that